native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/common/states/Http2StateUtil.java

```diff
@@ -282,7 +281,7 @@ public static void releaseDataFrame(Http2SourceHandler http2SourceHandler, Http2
     }
 
     /**
-     * Sends {@link Http2Reset} frame with `NO_ERROR` error code.
+     * Sends {@link RSTStream} frame with `NO_ERROR` error code.
```

Comment:
```suggestion
     * Sends `RST_STREAM` frame with `NO_ERROR` error code.
```

Reply:
Done in a3e217ee8a1c4b6e39d8b16a30e86b907c540379.